### DISCUSSION TITLE
Open a File?

### DISCUSSION THREAD

**Discussion by dlwiii** [tareqimbasher/NetPad#158] [Q&A]
Is it possible to open an existing file?  I simply do not see how. Usually File->Open would do it.

**Comment by tareqimbasher** [comment]
@dlwiii there is no `File > Open` yet, it will be added soon though.

**Reply by dlwiii** [reply]
How are you supposed to use it now?  Just type in fresh code every time?

**Reply by tareqimbasher** [reply]
@dlwiii currently NetPad will show you all scripts you've saved in its library folder. So if you save a script in the library folder it will be there the next time you open NetPad. Even if you don't save a script, unsaved changes are remembered and the next time you open NetPad the code you were working on will be there.

There isn't a way to open a file from somewhere not in the library folder yet. I confess that is such a basic feature and its missing... and it should be there. No reason really besides prioritizing other features. I'll prioritize this, but for now, if you have a script that is outside the library folder, just move it to the library folder and then it'll show up in the Explorer pane in NetPad and you can click on it to open it.

The library folder is typically in your `Documents` folder. On Windows for instance, that would be: `%USERPROFILE%\Documents\NetPad\Scripts`
